doc/how_to/migrate/from_streamlit.md

```diff
@@ -0,0 +1,627 @@
+# Migrate from Streamlit
+
+This guide addresses how to migrate from Streamlit to Panel via examples.
+
+This how-to guide can also be used as
+
+- an alternative *introduction to Panel* guide if you are already familiar with Streamlit.
+- a means of comparing Streamlit and Panel.
+
+---
+
+## Hello World
+
+Lets start by converting a *Hello World* application.
+
+### Streamlit Hello World Example
+
+In Streamlit a *Hello World* application looks like
+
+```python
+import streamlit as st
+
+st.write("Hello World")
+```
+
+You *serve* and *show* the app with *autoreload* via
+
+```bash
+streamlit run app.py
+```
+
+### Panel Hello World Example
+
+In Panel this looks like
+
+```python
+import panel as pn
+
+pn.extension(sizing_mode="stretch_width", template="bootstrap")
+
+pn.panel("Hello World").servable()
+```
+
+We use `pn.extension` to configure your application.
+
+- We set the `sizing_mode` to `stretch_width` to get the *responsive* behaviour you expect from
+Streamlit apps.
+- We set the `template` to `bootstrap` to wrap your application in a nice template as you expect
+from Streamlit apps. If you don't set a template you will get a blank template to add your
+components to. You can see the available templates in the
+[Templates Section](https://panel.holoviz.org/reference/index.html#templates) of the
```

Comment:
I believe we prefer not to link to the website directly, you should be able to see how it's done in other guides.